include/zephyr/bluetooth/audio/audio.h

```diff
@@ -845,6 +845,34 @@ struct bt_audio_stream {
 	void *user_data;
 };
 
+/** Unicast Client callback structure */
+struct bt_audio_unicast_client_cb {
+	/** @brief Remote Unicast Server Audio Locations
+	 *
+	 *  This callback is called whenever the audio locations is read from
+	 *  the server or otherwise notified to the client when changed.
```

Comment:
Nitpicking: The callback is called on notifications, independent of whether the notified value has changed or not.  (The _client_ is not checking whether the values has changed.)
(The fact that the _server_ probably only will notify on changes is irrelevant here.)
Propose to remove the " when changed".

Reply:
Good point :)